tests/behat/features/content-reviews.feature

```diff
@@ -39,6 +43,8 @@ Feature: Set up reviews
     When I press the "Save" button
     And I select "ADMIN group" from "Groups"
     And I press the "Save" button
+    And I select "EDITOR" from "Groups"
+    And I press the "Save" button
```

Comment:
I don't understand why we need to save two groups here?

Reply:
We have a button `.content-review__button` that appears only if user has special permission. So I add `EDITOR` to check this button in the same case.

Reply:
I removed `ADMIN group`.